CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+## v1.9.0
+
+This release focuses on bug fixes and developer tooling (e.g. vagrant).
+
+#### Changes
+
+- stage1: update coreos image to 1068.0.0 ([#2821](https://github.com/coreos/rkt/pull/2821)). This introduces support for arm64 and aligns to the current base of
```

Comment:
We typically just take the PR name as summary here (we probably should amend the changelog script to cut after the first line). Can you please drop all these half-sentences after the PR links?